Scripts/Chapter.js

```diff
@@ -1,4 +1,5 @@
 "use strict";
+var chapWidth = 0;
```

Comment:
You need to add a short comment saying what this variable is used for so it can be easily understood on a first read.